tests/phpunit/StorageExtractorTest.php

```diff
@@ -175,15 +155,14 @@ public function testTableInvalidFilter(): void
             'parameters' => [
                 '#token' => getenv('KBC_TEST_TOKEN'),
                 'url' => getenv('KBC_TEST_URL'),
-                'tables' => ['non-existent-table'],
             ],
         ];
         $baseDir = $temp->getTmpFolder();
         $fs->dumpFile($baseDir . '/config.json', \GuzzleHttp\json_encode($configFile));
         putenv('KBC_DATADIR=' . $baseDir);
         $app = new Component(new NullLogger());
-        self::expectExceptionMessage('The table non-existent-table not found in the bucket in.c-ex-storage');
         self::expectException(UserException::class);
+        self::expectExceptionMessage('The tableName parameter must be provided.');
```

Comment:
A tady uvozovky?

Reply:
nevim, je to nazev parametru, ne user provided value

Reply:
Souhlas. Kdybych to psal do zprávy, tak napíšu takhle, tak proto mi tam chybí...:

> The `tableName` parameter must be provided